chains/ethereum/package.json

```diff
@@ -17,12 +17,14 @@
     "typecheck": "tsc --noEmit"
   },
   "dependencies": {
-    "@coinbase/wallet-sdk": "3.5.2",
-    "@wagmi/core": "0.5.6",
+    "@coinbase/wallet-sdk": "3.5.3",
+    "@wagmi/core": "0.0.0-20220928064342",
```

Comment:
Not going to break?

Reply:
Should be good. It's based on 0.5.7 release and implements a fix that we discussed with wagmi team, which will be part of next release.